src/styles/calendar.scss

```diff
@@ -0,0 +1,22 @@
+@import './variables.scss';
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: $gap-size;
```

Comment:
Ensure that `$gap-size` is defined in your `variables.scss` file. Undefined variables will cause compilation errors.